CHANGELOG.md

```diff
@@ -13,10 +13,13 @@ Subheadings to categorize changes are `added, changed, deprecated, removed, fixe
 ### changed
 
 - Updated `usvg` to 0.41
+- MPL 2.0 is no longer a license requirement
+- The root image viewBox clipping was removed, to be added back at a later time
```

Comment:
Maybe this should be in the "Removed" section?

Reply:
Done